docs/user/power/install_upi.md

```diff
@@ -20,15 +20,15 @@ The steps for performing a UPI-based install are outlined here. Example automati
 
 ## Compute
 
-The smallest OpenShift 4.x clusters require the following host:
+The smallest OpenShift 4.x cluster requires the following hosts:
 
 * 1 bootstrap machine.
 
 * 3 control plane machines.
 
-* atleast 1 worker machine.
+* 2 worker machines.
```

Comment:
@manojnkumar please take a look at this comment. https://github.com/openshift/installer/pull/3454#discussion_r411662675
1 worker node is fine and the minimum required. However lets go with what @wking recommends

Reply:
@wking : Thoughts on the 1 worker vs 2 worker minimum?

Reply:
openshift-docs say 2